pixel_buffer::ellipse can't be compiled in MSVC

The method `void pixel_buffer::ellipse(const nana::point& pt, float a, float b, const color& c)` was added in commit bf26f622e382383969ce00c729b7ee30f6671f8b, and it breaks compilation under MSVC.

Compile log as follows:
```
[...]source\paint\pixel_buffer.cpp(1564,15): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data 
[...]source\paint\pixel_buffer.cpp(1565,15): warning C4244: 'initializing': conversion from 'float' to 'int', possible loss of data 
[...]source\paint\pixel_buffer.cpp(1568,18): error C2398: Element '2': conversion from 'float' to 'int' requires a narrowing conversion 
[...]source\paint\pixel_buffer.cpp(1568,18): warning C4244: 'argument': conversion from 'float' to 'int', possible loss of data 
[...]source\paint\pixel_buffer.cpp(1569,18): error C2398: Element '2': conversion from 'float' to 'int' requires a narrowing conversion 
[...]source\paint\pixel_buffer.cpp(1569,18): warning C4244: 'argument': conversion from 'float' to 'int', possible loss of data 
[...]source\paint\pixel_buffer.cpp(1572,12): warning C4244: '=': conversion from 'float' to 'int', possible loss of data 
[...]source\paint\pixel_buffer.cpp(1574,20): warning C4244: '=': conversion from 'float' to 'int', possible loss of data 
[...]source\paint\pixel_buffer.cpp(1599,15): error C2398: Element '1': conversion from 'float' to 'int' requires a narrowing conversion 
[...]source\paint\pixel_buffer.cpp(1599,15): warning C4244: 'argument': conversion from 'float' to 'int', possible loss of data 
[...]source\paint\pixel_buffer.cpp(1600,15): error C2398: Element '1': conversion from 'float' to 'int' requires a narrowing conversion 
[...]source\paint\pixel_buffer.cpp(1600,15): warning C4244: 'argument': conversion from 'float' to 'int', possible loss of data 
[...]source\paint\pixel_buffer.cpp(1602,12): warning C4244: '=': conversion from 'float' to 'int', possible loss of data 
[...]source\paint\pixel_buffer.cpp(1604,20): warning C4244: '=': conversion from 'float' to 'int', possible loss of data
```

Basically, `a` and `b` need to be rounded before being added to `x` and `y`, respectively.

--- a/source/paint/pixel_buffer.cpp
+++ b/source/paint/pixel_buffer.cpp
@@ -1561,17 +1561,17 @@ namespace nana{	namespace paint
 		auto col = c.px_color();
 
 		    int wx, wy;
-		    int asq = a * a;
-		    int bsq = b * b;
+		    int asq = int(a * a);
+		    int bsq = int(b * b);
 		    int xa, ya;
 
-		    pixel({x, y+b}, col);
-		    pixel({x, y-b}, col);
+		    pixel({x, y + (int)b}, col);
+		    pixel({x, y - (int)b}, col);
 
 		    wx = 0;
-		    wy = b;
+		    wy = (int) b;
 		    xa = 0;
-		    ya = asq * 2 * b;
+		    ya = asq * 2 * (int) b;
 		    auto thresh = asq / 4 - asq * b;
 
 		    for (;;) {
@@ -1596,12 +1596,12 @@ namespace nana{	namespace paint
 		        pixel({x-wx, y+wy}, col);
 		    }
 
-		    pixel({x+a, y}, col);
-		    pixel({x-a, y}, col);
+		    pixel({x + (int) a, y}, col);
+		    pixel({x - (int) a, y}, col);
 
-		    wx = a;
+		    wx = (int) a;
 		    wy = 0;
-		    xa = bsq * 2 * a;
+		    xa = bsq * 2 * (int) a;
 
 		    ya = 0;
 		    thresh = bsq / 4 - bsq * a;

